src/Components/Admins/Form/admins-form.module.css

```
@@ -0,0 +1,25 @@
+.sectionForm {
+    margin: 40px 0;
+}
+
+.form {
+    color: #373867;
+    background-color: #eee;
+    padding: 25px;
+    border-radius: 8px;
+    width: 400px;
+    margin: 10vh auto;
+}
+
+.subContainer {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 10px;
+}
+
+
+.btnContainer {
+    width: 85%;
```

all the forms has 100% here
```suggestion
    width: 100%;
```